sinopia_client/docs/LDPApi.md

```diff
@@ -45,11 +45,9 @@ Create a new Group (defined via JSON-LD in payload) within the base container.
 var SinopiaServer = require('sinopia_server');
 var defaultClient = SinopiaServer.ApiClient.instance;
 
-// Configure API key authorization: RemoteUser
-var RemoteUser = defaultClient.authentications['RemoteUser'];
-RemoteUser.apiKey = 'YOUR API KEY';
-// Uncomment the following line to set a prefix for the API key, e.g. "Token" (defaults to null)
-//RemoteUser.apiKeyPrefix = 'Token';
+// Configure OAuth2 access token for authorization: CognitoUser
+var CognitoUser = defaultClient.authentications['CognitoUser'];
+CognitoUser.accessToken = 'YOUR ACCESS TOKEN';
```

Comment:
another very similar approach that you could use if you got an `ApiClient` instance like in the specs:
```javascript
instance = new SinopiaServer.LDPApi()
instance.apiClient.authentications['CognitoUser'].accessToken = 'AN_ENCODED_JWT'
// same idea as the way the tests set the server URL:
// instance.apiClient.basePath = 'http://localhost:8080'
```